### PR TITLE
feat(monitor): Add a chart that starts the deis monitoring stack

### PR DESCRIPTION
**Pull request by jchauncey**
Also adds the monitoring charts to workflow-dev

### CODE REVIEW

File: deis-monitor/manifests/deis-monitor-grafana-rc.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    spec:
+      containers:
+      - name: deis-monitor-grafana
+        image: quay.io/deis/grafana:canary
+        imagePullPolicy: Always
+        env:
+        - name: "INFLUXDB_URLS"
+          value: http://$(DEIS_MONITOR_INFLUXAPI_SERVICE_HOST):$(DEIS_MONITOR_INFLUXAPI_SERVICE_PORT_TRANSPORT)
+        - name: "KUBERNETES_DOCKER_ENDPOINT"
+          value: "unix:///var/run/docker.sock"
```
Comment (arschles) [inline deis-monitor/manifests/deis-monitor-grafana-rc.yaml]: would it make sense to parameterize this so clusters who don't use this socket can configure this chart as necessary?

**Comment by arschles** [comment]
I'm not sure if https://github.com/deis/charts/pull/248#discussion_r62558538 is blocking for this PR - figuring you'd know better than me, so I'm LGTMing this

**Comment by jchauncey** [comment]
@arschles im not sure why that is there. testing relaly quick without it. I think thats a hold over from a long time ago

**Comment by jchauncey** [comment]
@arschles removed wasnt needed. sorry bout that good catch

**Pull request merged**